src/components/create-deck/create-deck.js

```diff
@@ -0,0 +1,121 @@
+import EditPanel from '../edit-panel/edit-panel';
+import { Component } from 'react';
+
+import './create-deck.css';
+
+const store = require('store');
+
+class CreateDeck extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
I'm confused with this state structure
try to implement this component using useState hook and separate logic for different elements